backend/tests/test_pe_csv_import.py

```diff
@@ -202,3 +206,33 @@ async def test_parse_principal_csv_exception(
         await import_beneficiaries(db_connection, pe_principal_csv_filepath)
 
         assert "Exception while processing CSV line: Parsing exception" in caplog.text
+
+
+async def test_email_conversion():
+    assert (
+        net_email_to_fr_email("051referent_prenom4.referent_nom4@pole-emploi.net")
+        == "referent_prenom4.referent_nom4@pole-emploi.fr"
+    )
+
+
+async def test_import_pe_referent(
+    db_connection: Connection, pe_principal_csv_series: DataFrame
+):
+    iterator = pe_principal_csv_series.iterrows()
+    for i, (_, row_content) in enumerate(iterator):
```

Comment:
ne serait-ce pas plus simple d'accéder directement au 4ème élément de la série ?

Reply:
C'est un itérateur et ce n'est pas possible en python d'accéder à l'élément X d'un itérateur